tests/framework/load.go

```diff
@@ -31,38 +31,54 @@ func convertTargetToVegetaTarget(targets []Target) []vegeta.Target {
 	return vegTargets
 }
 
+// Metrics is a wrapper around the vegeta Metrics.
+type Metrics struct {
+	vegeta.Metrics
+}
+
 // RunLoadTest uses Vegeta to send traffic to the provided Targets at the given rate for the given duration and writes
 // the results to the provided file
 func RunLoadTest(
 	targets []Target,
```

Comment:
Would it make sense to introduce a config type, because the number of parameters is growing?
Since some of them are of the same type, like consecutive `desc, proxy, serverName`, it becomes a bit error prone and easy to get the order wrong.  Having a config type will prevent that, since we will use the name of the fields of that type.